doctrine/use_enum.rst

```diff
@@ -0,0 +1,43 @@
+Use Enums
+=========
+
+PHP 8.1 added native Enumerations, which can be used to define a custom type
+and its possible values. Those can be used in association with Doctrine in
+order to define a limited set of values availables for an entity property.
+
+First step is to create an enum::
+
+    // src/Enum/Suit.php
+    namespace App\Enum;
+
+    enum Suit: string {
+    case Hearts = 'H';
+    case Diamonds = 'D';
+    case Clubs = 'C';
+    case Spades = 'S';
+    }
+
+.. note::
+
+    Only backed enums can be used with properties as Doctrine use the scalar
+    equivalent of each value for storing.
+
+When the enum is created, you can use the ``enumType`` parameter of
+``#[ORM\Column]`` attribute or use it directly for a more typed property::
+
+    // src/Entity/Card.php
+    namespace App\Enum;
+
+    #[Column(type: Types::TEXT, enumType: Suit::class)]
+    public string $suit;
```

Comment:
It's a rare use case to map a property using a BackedEnum and not use it in the model; I wouldn't show this case. I think the above example should be the only one documented.

Reply:
I see your point. I'm quite in favour of showing both even if one use is rare and let developers choose the one that fit to their needs. Maybe we can add a comment to point which is the most common/recommended way ?